src/org/stringtemplate/v4/Interpreter.java

```diff
@@ -695,6 +695,31 @@ protected int writeObjectNoOptions(STWriter out, InstanceScope scope, Object o)
         return n;
     }
 
+    /** Write out a text element, i.e. a part of the template that is neither expression nor comment
+     */
+    protected int writeText(STWriter out, InstanceScope scope, String o) {
+        int start = out.index(); // track char we're about to write
+        int n = writeTextObject(out, scope, o);
+        if ( debug ) {
+            EvalExprEvent e = new EvalExprEvent(scope,
```

Comment:
I wonder whether there should be a different event for this. After all, text elements are not considered to be expressions, so `EvalExprEvent` may be misleading.